package.json

```diff
@@ -23,19 +23,19 @@
     "update:interactive": "npx npm-check -uE"
   },
   "devDependencies": {
-    "@babel/core": "7.14.6",
-    "@babel/eslint-parser": "7.14.7",
-    "eslint": "7.30.0",
-    "eslint-plugin-jsdoc": "35.4.1",
-    "eslint-plugin-unicorn": "34.0.1",
-    "lint-staged": "11.0.0",
-    "simple-git-hooks": "2.5.1"
+    "@babel/core": "7.16.0",
+    "@babel/eslint-parser": "7.16.3",
+    "eslint": "8.4.0",
+    "eslint-plugin-jsdoc": "37.1.0",
+    "eslint-plugin-unicorn": "39.0.0",
+    "lint-staged": "12.1.2",
+    "simple-git-hooks": "2.7.0"
   },
   "peerDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/eslint-parser": "^7.0.0",
-    "eslint": "^7.0.0",
-    "eslint-plugin-jsdoc": ">=35.0.0",
-    "eslint-plugin-unicorn": ">=34.0.0"
+    "eslint": "^8.0.0",
```

Comment:
Én `eslint` in de `devDependencies` én in de `peerDependencies`? 🤔

Reply:
Hm hm, de een is voor dit project, de andere voor de projecten waar `eslint` als dependency wordt geinstalled.